addon/components/resize-sensor.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+
+import ResizeSensor from 'resize-sensor';
+
+export default Component.extend({
+  classNames: ['resize-sensor-container'],
+
+  didInsertElement() {
+    const element = this.element;
+    this._resizeSensor = new ResizeSensor(element, () => {
+      const onResize = this.get('onResize');
```

Comment:
We should wrap the callback in `run.next` so it triggers a runloop, otherwise we don't have any guarantee when this action will be called